### PR TITLE
Added the Nodes locally hosted service listing to sysinfo.json

### PR DESCRIPTION
**Pull request by r1de**
Added the Nodes locally hosted service listing to sysinfo.json file.

Yes, another addition for my mapping system.
The only way to get this info before was port 9090 and we know how that went. :)

### CODE REVIEW

File: files/www/cgi-bin/sysinfo.json [hunk 1]
```diff
@@ -244,6 +244,33 @@ if string.find(nixio.getenv("QUERY_STRING"):lower(),"services=1") then
 	info["services"]=services
 end
 
+-- LOCALLY HOSTED SERVICES ONLY
+if string.find(nixio.getenv("QUERY_STRING"):lower(),"services_local=1") then
```
Comment (dman776) [inline files/www/cgi-bin/sysinfo.json:248]: I would prefer "local_services" instead in order to match the info[] table name

Comment (r1de) [reply files/www/cgi-bin/sysinfo.json:248]: I get what you are saying now. Sorry. :smile:
I made this "services_local=1" due to the fact that "local_services=1" will *also* trigger the "services=1" function.
I wasn't looking for *all* of the services, just the locally hosted ones, and felt that triggering the other function at the same time was a big waste of bandwidth.
I am open to other names for it. I just used those so they would make sense.

Comment (r1de) [reply files/www/cgi-bin/sysinfo.json:248]: I could just swap the info[] line to read services_local instead. :smile:

File: files/www/cgi-bin/sysinfo.json [hunk 2]
```diff
@@ -244,6 +244,33 @@ if string.find(nixio.getenv("QUERY_STRING"):lower(),"services=1") then
...
+				service['protocol']=protocol
+				service['link']=link
+				table.insert(lclsrvs, service)
+			end
+		end
+	else
+		service['error']="Cannot read local services file"
+		table.insert(lclsrvs, service)
+	end
+	info['local_services']=lclsrvs
```
Comment (dman776) [inline files/www/cgi-bin/sysinfo.json]: agree, for now, just rename the info table to info['services_local']

**Comment by r1de** [comment]
Done

**Pull request merged**